cob_twist_controller/src/constraint_solvers/solvers/stack_of_tasks_solver.cpp

```diff
@@ -106,12 +106,23 @@ Eigen::MatrixXd StackOfTasksSolver::solve(const Vector6d_t& in_cart_velocities,
         Eigen::MatrixXd J_temp = J_task * projector_i;
         Eigen::VectorXd v_task = it->task_;
         Eigen::MatrixXd J_temp_inv = pinv_calc_.calculate(J_temp);
+        Eigen::FullPivLU<Eigen::MatrixXd> lu_decomp(J_temp_inv);
```

Comment:
Because J_temp_inv is going to project the ith task velocities. If it has rank zero is not adding any velocity for that task.